service/src/main/resources/api/service_openapi.yaml

```diff
@@ -2913,6 +2913,8 @@ components:
           $ref: '#/components/schemas/JobReport'
         gcpContext:
           $ref: '#/components/schemas/GcpContext'
+        azureContext:
```

Comment:
This is an additive API modification to allow for the inclusion of an azure cloud context in a cloud context result response body.